src/s2a_moms_tent.tw2

```diff
@@ -0,0 +1,287 @@
+
+I wanted to scream, shake her, yell at her, do something. But the moment passed, and instead I felt [[tears welling up my eyes|Mom's Tent - letter to mom start]].
+
+<!-- If using debug menu shortcuts, prompts player to enter their name -->
+::Mom's Tent - name prompt letter
+{ (set: $playerName to (prompt: "Enter name:", ""))
+(if: (text: $playerName)'s length > 0)[Right, my name is $playerName. [[Moving on|Mom's Tent - letter to mom start]].]
+}(else:)[No, [[that's not right|Mom's Tent - name prompt letter]].]
+
+::Mom's Tent - letter to mom start
+Why the hell would she fall asleep in my bed? She’s got her own, and it’s way nicer.
+
+I couldn’t just leave. I needed to at least let her know what’s up.
+
+I turned back to my desk, picked up a pencil, and began to write:
+
+>>(link: "Hey,")[(set: $letterHey to true)(go-to: "Mom's Tent - letter to mom 2")]
+>>[[Hey Mom,|Mom's Tent - letter to mom 2]]
+
+::Mom's Tent - letter to mom 2
+(if: $letterHey is true)[Hey,](else:)[Hey Mom,]
+
+>>(link:"I’m sorry I didn’t make it home last night. Things are messed up, but it’s not my fault.")[(set: $letterApology to true)(go-to: "Mom's Tent - letter to mom 3")]
+>>[[I came home to get some rest, but it turns out my bed was occupied. Thanks a lot.|Mom's Tent - letter to mom 3]]
+
+::Mom's Tent - letter to mom 3
+(if: $letterHey is true)[Hey](else:)[Hey Mom,]
+
+(if: $letterApology is true)[I’m sorry I didn’t make it home last night. Things are messed up, but it’s not my fault.](else:)[I came home to get some rest, but it turns out my bed was occupied. Thanks a lot.]
+
+>>(link:"I’m worried about Anna. She slipped me something and straight-up disappeared on me last night. It’s not like her at all. I’m worried shit’s gone way wrong for her.")[(set: $letterWorried to true)(go-to: "Mom's Tent - letter to mom 4")]
+>>[[I just dropped by to get some of my stuff.|Mom's Tent - letter to mom 4]]
+
+::Mom's Tent - letter to mom 4
+(if: $letterHey is true)[Hey](else:)[Hey Mom,]
+
+(if: $letterApology is true)[I’m sorry I didn’t make it home last night. Things are messed up, but it’s not my fault.](else:)[I came home to get some rest, but it turns out my bed was occupied. Thanks a lot.]
+
+(if: $letterWorried is true)[I’m worried about Anna. She slipped me something and straight-up disappeared on me last night. It’s not like her at all. I’m worried shit’s gone way wrong for her.](else:)[I just dropped by to get some of my stuff.]
+
+>>(link:"I know Anna’s in trouble, and I’ve gotta make sure she’s ok.")[(set: $letterTrouble to true)(go-to: "Mom's Tent - letter to mom 5")]
+(if $beerDrank is true)[>>[[I drank some of Ron’s Milenkale. Be sure to let him know it was me.|Mom's Tent - letter to mom 5]]](else:)[>>[[I drank some of Ron’s Faygo. Be sure to let him know it was me.|Mom's Tent - letter to mom 5|Mom's Tent - letter to mom 5]]]
```

Comment:
Think I found it! You're missing a colon after the `if`.

Reply:
Also, I noticed at the end of this line, the Faygo option has `|Mom's Tent - letter to mom 5|Mom's Tent - letter to mom 5` repeated. Make sure to take out the second instance, and everything should work perfectly 👌

Reply:
Oh my god. Thank you! 🙏 I knew it had to be something like this, but when I finished this my brain and eyes were no longer able to pick these details out. Coding exhaustion is no joke.